packages/jupyter-ai/jupyter_ai/models.py

```diff
@@ -29,13 +29,32 @@ class ChatClient(ChatUser):
     id: str
 
 
+class PersonaDescription(BaseModel):
```

Comment:
These are good suggestions, thank you! For context, I'm following the naming conventions used by AWS APIs. I prefer "description" because it implies that the `PersonaDescription` model is smaller than the `Persona` model, in the same way the description of a book is shorter than the book itself.

- The name "PersonaDetails" implies that this model contains additional details on top of the `Persona` model.
- The name "PersonaModel" doesn't distinguish itself from `Persona` enough, since "model" is already an overused term in this source repo.

I'm still open to other suggestions; I think that `PersonaDescription` is the best name here however.